### ISSUE TITLE
check_files on UNC path

### ISSUE THREAD

**Issue by willemdh** [mickem/nscp#194]
I'm trying to alert if the file count on an UNC path exceeds 0. 
```
/usr/local/nagios/libexec/check_nrpe -H server -c check_files -a path='\\domain\Junction\NFI_OG\PR\DebiteurenBoeking\Error' pattern=*.* 'critical=count>0'
No drives found|'count'=0;0;0
```
The nscp service is runnign as an account with permissions on the unc path. I keep getting "No drives found"
Is check_files compatible with UNC paths?

**Comment by mickem** [comment]
no drives found is a copy paste issue, should most likely read no files found...so it seems "to work" but I know errors are in some cases reported on the nsclient++ log so might wanna check that...

**Comment by willemdh** [comment]
Ah yes, it does indeed work. Sorry, when I read 'No drives found' I thought it didn't found the path. When I add a testfile and execute:
````
/usr/local/nagios/libexec/check_nrpe -H server -c check_files -a path='\\domain\Junction\NFI_OG\PR\DebiteurenBoeking\Error' pattern=*.* 'critical=count>0'
```
```
CRITICAL: 0/1 files (test.txt)|'count'=1;0;0
```
What ddoes it mean 0/1? When I put two files there it says 0/2. Why the zero/ ?

**Comment by mickem** [comment]
The default value for details-syntax is: top-syntax	${status}: ${problem_count}/${count} files (${problem_list})
See https://docs.nsclient.org/0.4.2/reference/CheckDisk.html#check-files for details.

So the first value is "number of files matching your filter" the second one is number of files.
As you check for count the default syntax makes less sense I guess, if your checking for file-size or some such you would get 3/4 files where the 3 would be number of files to large or whatever.

Please note that count is broken in <0.4.4 it wont work with less then i.e. "crit=count < 4" will not work...

**Comment by mickem** [comment]
Sorry... "number of files matching your filter" should be "number of files matching your crit/warn"

**Comment by willemdh** [comment]
Ok, Thanks a lot Michael. Looking forward to a stable release of 0.4.4+

**Comment by willemdh** [comment]
Hmm I have something weird. Nothing really changed. The server I'm running this check on is actually on 5.0.18. All of a sudden

    -H gateway -t 60 -c check_files -a path='\\stadgent\Junction\NFI_OG\PR\DebiteurenBoeking\Error' pattern=*.* 'critical=count>0'

Results in an unknown state. There are no files in the location \\stadgent\Junction\NFI_OG\PR\DebiteurenBoeking\Error

Beore yesterday when there are no files I got an OK. When I put a file in the \\stadgent\Junction\NFI_OG\PR\DebiteurenBoeking\Error folder I do gat a critical.

Afetr checking the gateway I found out that there are actually 2 installs of NSClient++ on the server..

![image](https://cloud.githubusercontent.com/assets/6462991/13245044/9bbdaf9e-da0a-11e5-9d84-6f46534a357e.png)

I'm further investigating this. Any idea what might be causing the sudden UNKNOWN state?
